### ISSUE TITLE
Rails without Asset Pipeline Bug: Still requires precompile

### ISSUE THREAD

**Issue by chrishough** [resque/resque-web#111]
As noted in comments on https://github.com/resque/resque-web/pull/38#issuecomment-233102724 it looks like this gem still requires assets to be pre-compiled locally if you are running rails without sprockets.  

In order to get resque web to run I had to add this to my environments/development.rb file:
```
# This is required for Resque Web
config.assets.compile = true
```
and this line to my initializers/resque.rb file:  
```
Rails.application.config.assets.precompile += %w(resque_web/*.png) 
```

**Comment by kirillplatonov** [comment]
@chrishough fixed in #112. Could you check it with your project please?

**Comment by chrishough** [comment]
ok. This works now locally, but it is borked on the server :( https://github.com/resque/resque-web/issues/117
